config/config.go

```diff
@@ -6,14 +6,28 @@ package config
 
 import (
 	"flag"
+	"os"
+	"path"
 )
 
 // All configuration values which should be available for use by other packages need to be defined here as global variables.
+// Any configuration that doesn't require custom default values can have their flags initialised right here.
+// Any configuration that require custom default values should be initalized manually.
 var (
 	LogLevel    = flag.String("loglevel", "debug", "Log messages upto this level")
 	RestAddress = flag.String("restaddress", ":24007", "Address to bind REST endpoint to")
+
+	LocalStateDir string
 )
 
+func initLocalStateDir() {
+	wd, _ := os.Getwd()
```

Comment:
Is it same to generate the UUID in the current working directory or we should have it in /var/lib/glusterd?